queries/snakemake/highlights.scm

```diff
@@ -33,6 +32,7 @@
   "use" @keyword.import
   "rule" @keyword.import
   "from" @keyword.import
+  "exclude"? @keyword.import
   "as"? @keyword.import
   "with"? @keyword.import)
```

Comment:
Can this be reduced to

```suggestion
[
  "use"
  "rule"
  "from"
  "exclude"
  "as"
  "with"
] @keyword.import
```

Reply:
The scoping to `(rule_import)` is required, but grouping is fine.